ogc-example/src/main/scala/geotrellis/server/ogc/wmts/WmtsView.scala

```diff
@@ -90,11 +82,7 @@ class WmtsService(
                 logger.error(err.toString, err)
                 InternalServerError(err.toString)
             }
-          }).getOrElse(BadRequest("No such layer"))
+          }).getOrElse(BadRequest("Layer not found"))
```

Comment:
Hm, It's not critical, but mb it makes sense to add layer name into the BadRequest response?